docs/hub/how-to-downstream.md

```diff
@@ -0,0 +1,76 @@
+# How to integrate downstream methods in your library
+
+In the tutorial, you learned some basic methods for integrating the Hub into your library. Utilities that allow your library to download files from the Hub are referred to as *downstream* utilities. This guide introduces additional downstream utilities you can integrate with your library. You will learn how to:
```

Comment:
Just in case these "methods" are also supposed to be "utilities" (ignore if not)
```suggestion
In the tutorial, you learned some basic utilities for integrating the Hub into your library. Utilities that allow your library to download files from the Hub are referred to as *downstream* utilities. This guide introduces additional downstream utilities you can integrate with your library. You will learn how to:
```

Reply:
Good catch, thanks!